### DISCUSSION TITLE
Two months after approval, I still haven't gotten my GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by LinLin00000000** [community/community#77000] [GitHub Education]
### Hi everyone,

I submitted my student verification information on October 1, and the request was quickly approved. But until now, I still haven't gotten the GitHub Student Developer Pack.

I really like Github, and I need to learn to program and create something interesting!

Can anyone help me?

![image](https://github.com/community/community/assets/51076594/069137fb-a924-45f4-b58c-f1102a14782f)
![image](https://github.com/community/community/assets/51076594/28ccffda-01ac-43af-ab72-0479119da82e)

**Comment by hackelite01** [comment]
Firstly, make sure to check your email, including your spam folder, as sometimes the GitHub Student Developer Pack details are sent there. If you still can't find it, try logging out of your GitHub account and logging back in. This simple step has resolved similar issues for some users.

If the problem persists, please don't hesitate to reach out to me at hackelie.sup@gmail.com.

**Comment by notsointresting** [comment]
I suggest that you check your email inbox (including the spam folder) for any communication from GitHub regarding your application. If you can't find any, you should contact GitHub Support directly. They will be able to provide you with more information about your application status. You can reach out to them through this [link](https://support.github.com/contact)

**Comment by LinLin00000000** [comment]
> I suggest that you check your email inbox (including the spam folder) for any communication from GitHub regarding your application. If you can't find any, you should contact GitHub Support directly. They will be able to provide you with more information about your application status. You can reach out to them through this [link](https://support.github.com/contact)

I'm sure there was nothing in the e-mail about the subject. I then contacted GitHub Support as you suggested, but it's been three weeks since I sent the work order and I haven't gotten any response, is Github dead?

**Comment by LinLin00000000** [comment]
> Firstly, make sure to check your email, including your spam folder, as sometimes the GitHub Student Developer Pack details are sent there. If you still can't find it, try logging out of your GitHub account and logging back in. This simple step has resolved similar issues for some users.
> 
> If the problem persists, please don't hesitate to reach out to me at [hackelie.sup@gmail.com](mailto:hackelie.sup@gmail.com).

I tried that. It doesn't work.

**Comment by LinLin00000000** [comment]
The GitHub Student Developer Pack isn't that important to me. I'm just curious, does Github not care about Chinese users?

**Comment by LinLin00000000** [comment]
![image](https://github.com/community/community/assets/51076594/af1debd0-68c3-489f-82ff-ac71e75801ef)

A new discovery, when I looked again today I realized that my request expired the day before yesterday, however I didn't receive any notification, and neither was it in my mailbox (including the spam folder). I'd like to ask Github what's going on here?

I have two years to go before I graduate, why does it show as expired after only two months? And I didn't get any student benefits at all during this process.

All of a sudden Approved and then Expired, ridiculous!

**Comment by LinLin00000000** [comment]
> If you're experiencing issues with obtaining your GitHub Student Developer Pack after the verification of your student status, there are several steps you can take to resolve this:
> 
> Contact GitHub Support: You can reach out to GitHub's support team to inquire about the status of your request for the Student Developer Pack. They may be able to provide you with specific information and help resolve any potential issues.
> 
> Double-Check Eligibility: Ensure that you meet all the eligibility requirements for the GitHub Student Developer Pack. It's important to confirm that you have provided all the necessary documentation and that your student status has been verified correctly.
> 
> Follow Up with Your Educational Institution: Sometimes, delays in providing access to student benefits can be related to the verification process at the educational institution. Check with your school's administration to ensure that the necessary steps have been completed from their end.
> 
> Explore Alternative Resources: While waiting for the GitHub Student Developer Pack, you can explore other resources and platforms for learning and creating projects. There are numerous free tutorials, open-source tools, and online communities that can help you develop your programming skills and work on interesting projects.
> 
> Engage with the Community: Don't hesitate to seek assistance from online developer communities, forums, or social media groups. You might find someone who has encountered a similar situation and can offer advice on how to proceed.

Do you only have the official standard answers and don't even look closely at the problems I'm having?
